.travis.yml

```diff
@@ -10,4 +10,7 @@ install:
 #command to run tests
 script: nosetests --exe --with-coverage --cover-package=music_feats
 
+#command to build sphinx documentations
+script: sphinx-build -nW -b html -c ./config -d _build/doctrees . _build/html
```

Comment:
For multiple lines in a specific keyword "script", try
`script:
  - nosetests --exe --with-coverage --cover-package=music_feats
  - sphinx-build -nW -b html -c ./config -d _build/doctrees . _build/html`
I'm actually not sure if this would work however...